lib/bokken_web/controllers/pairing_controller.ex

```diff
@@ -4,17 +4,15 @@ defmodule BokkenWeb.PairingController do
   alias Bokken.Events
   alias Bokken.Pairings
 
-  defguard is_organizer(conn) when conn.assigns.current_user.role === :organizer
-
-  def index(conn, %{"event_id" => event_id}) when is_organizer(conn) do
+  def index(conn, %{"event_id" => event_id}) do
     lectures = Events.get_lectures_from_event(event_id)
 
     conn
     |> put_status(:ok)
     |> render("index.json", lectures: lectures)
   end
 
-  def create(conn, %{"event_id" => event_id}) when is_organizer(conn) do
+  def create(conn, %{"event_id" => event_id}) do
     lectures = Pairings.create_pairings(event_id)
 
     conn
```

Comment:
This controller should only be able to an organizer. So that, put the `is_organizer/1` guard back 🙏🏼